.env.production

```
@@ -0,0 +1,222 @@
+# ============================================================================
+TRIPO3D_API_KEY=REPLACE_WITH_PRODUCTION_KEY
+FASHN_API_KEY=REPLACE_WITH_PRODUCTION_KEY
+GOOGLE_IMAGEN_API_KEY=REPLACE_WITH_PRODUCTION_KEY
+
+# ============================================================================
+# WORDPRESS/WOOCOMMERCE INTEGRATION (Production Site)
+# ============================================================================
+# Production URL: https://skyyrose.co
+WORDPRESS_URL=https://skyyrose.co
+WORDPRESS_USERNAME=REPLACE_WITH_ADMIN_USERNAME
+WORDPRESS_APP_PASSWORD=REPLACE_WITH_APP_PASSWORD
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The WORDPRESS_APP_PASSWORD key should go before the WORDPRESS_URL key

(UnorderedKey)